pkg/cluster/cluster.go

```diff
@@ -760,6 +763,30 @@ func (c *Cluster) initHumanUsers() error {
 	return nil
 }
 
+func (c *Cluster) initHumanUsers() error {
+
+	for _, postgresSuperuserTeam := range c.OpConfig.PostgresSuperuserTeams {
+		err := c.initTeamMembers(postgresSuperuserTeam, true /*isPostgresSuperuserTeam*/)
+		if err != nil {
+			return fmt.Errorf("Cannot create a team %q of Postgres superusers: %v", postgresSuperuserTeam, err)
+		}
+	}
+
+	for _, postgresSuperuserTeam := range c.OpConfig.PostgresSuperuserTeams {
+		if postgresSuperuserTeam == c.Spec.TeamID {
+			c.logger.Infof("Team %q owning the cluster is also a team of superusers. Created superuser roles for its members instead of admin roles.", c.Spec.TeamID)
+			return nil
+		}
+	}
+
+	err := c.initTeamMembers(c.Spec.TeamID, false /*isPostgresSuperuserTeam*/)
```

Comment:
Seems this can be skipped if superuser team contains team already? (hinted by Oleksii)

Reply:
it is skipped in the lines ahead